src/test/java/com/google/api/codegen/testdata/nodejs_doc_main_library.baseline

```diff
@@ -115,13 +115,42 @@ function LibraryServiceClient(gaxGrpc, grpcClients, opts) {
 
   var bundleDescriptors = {
     publishSeries: new gax.BundleDescriptor(
-        'books',
-        [
-          'edition',
-          'shelf.name'
-        ],
-        'bookNames',
-        gax.createByteLengthFunction(grpcClients.libraryServiceClient.google.example.library.v1.Book))
+      'books',
+      [
+        'edition',
+        'shelf.name'
+      ],
+      'bookNames',
+      gax.createByteLengthFunction(grpcClients.libraryServiceClient.google.example.library.v1.Book))
+  };
+
+  var operationsClient = new gax.lro({
+    servicePath: servicePath,
+    port: port,
+    sslCreds: sslCreds,
+    clientConfig: clientConfig,
+    appName: appName,
+    appVersion: appVersion
+  });
+
+  var protoDescriptorPool = {};
+
+  extend(protoDescriptorPool, gaxGrpc.load([{
+    root: require('google-proto-files')('..'),
+    file: 'library.proto'
+  }]));
+
+  extend(protoDescriptorPool, gaxGrpc.load([{
+    root: require('google-proto-files')('..'),
+    file: 'tagger.proto'
+  }]));
```

Comment:
Also, this is quite similar to `grpcClients` structure. It would be better to consolidate them into a single construct, so that it does not have to load/parse the proto twice.